<!-- -*- mode: xml -*- -->
<launch>
  <!-- -->
  <arg name="vision_machine" default="carola" />
  <machine name="vision" address="$(arg vision_machine)" />
  <!-- -->
  <group ns="HRP2016V">
  <include file="$(find cr_capture)/launch/camera1394_hrp2016v.launch"/>
  <include file="$(find cr_capture)/launch/swissranger_sr4000.launch"/>
  <include file="$(find cr_capture)/launch/cr_capture.launch"/>
  <!-- <include file="$(find cr_capture)/launch/robot_self_filter.launch"/> -->
  <node pkg="jskpointcloud"
        name="$(anon remove_outlier)"
        type="remove_outlier"
        clear_params="true">
    <remap from="points_old_in"
           to="/HRP2016V/cr_capture/color_pcloud" />
    <remap from="points_out"
           to="/HRP2016V/cr_capture/color_pcloud_cleared" />
    <param name="mean_k" value="70" />
    <param name="mul_thresh" value="2.0" />
  </node>
  </group>
</launch>
